### DISCUSSION TITLE
We need ignore duplicate write Option for bulkWrite

### DISCUSSION THREAD

**Discussion by gookseangkim** [openfga/community#385] [Ideas]
We need ignore duplicate write Option for bulkWrite.
If we write tuple and the tuple already exist in database, we got an error for "duplicate tuple in write".
for the case write 1 by 1 we can verify exist before writing, but in bulkWrite case, we can not ensure whether the tuple exist in this bulk writing tuples, I think we need an option to ignore the error for "duplicate tuple in write" in bulk write

**Comment by aaguiarz** [comment]
This was implemented
